code/game/gamemodes/wizard/godhand.dm

```diff
@@ -28,14 +34,10 @@
 /obj/item/melee/touch_attack/afterattack(atom/target, mob/user, proximity)
 	if(catchphrase)
 		user.say(catchphrase)
-	playsound(get_turf(user), on_use_sound,50,1)
-	attached_spell.attached_hand = null
-	qdel(src)
-
-/obj/item/melee/touch_attack/Destroy()
+	playsound(get_turf(user), on_use_sound, 50, 1)
 	if(attached_spell)
-		attached_spell.attached_hand = null
-	return ..()
+		attached_spell.perform(new /list)
```

Comment:
```suggestion
		attached_spell.perform(list())
```
I think this works just aswell? (unless I'm wrong)

Reply:
updated